packages/corvid-fake-local-mode-editor/src/editor.js

```diff
@@ -115,7 +115,8 @@ const getSiteDocumentFromServer = async socket =>
 const loadEditor = async (
   port,
   { siteDocument: initialSiteDocument, siteCode: initialSiteCode } = {},
-  { cloneOnLoad = true } = {}
+  { cloneOnLoad = true } = {},
+  extraArgs = {}
```

Comment:
we already have an options param (the one cloneOnLoad is in)

Reply:
done